### PR TITLE
Feat/add ai page to homepage

### PR DESCRIPTION
**Pull request by tomwisecodes**
- Adds a new tab to the home page tabs for ai.
- Adds a new hero for that tab linking out to the labs.thenational.academy 
- Adds labs as a possible route for oak pages

### CODE REVIEW

**Review by mcky — APPROVED** [review]
Approved code-wise **PENDING** a decision about the analytics from the team

File: src/pages/index.tsx [hunk 1]
```diff
@@ -47,7 +48,7 @@ export type HomePageProps = {
   posts: SerializedPost[];
 };
 
-export type HomePageTab = "teachers" | "curriculum" | "pupils";
+export type HomePageTab = "teachers" | "curriculum" | "pupils" | "ai";
 const isHomePageTab = (u: unknown): u is HomePageTab =>
```
Comment (kpopper) [inline src/pages/index.tsx:52]: Does `isHomePageTab` need to be set if the tab is `ai`?

Comment (rightsaidjames) [reply src/pages/index.tsx:52]: I think this is needed to take the user back to the AI tab when they press the back button from another page.

Comment (tomwisecodes) [reply src/pages/index.tsx:52]: Good flag, I think this persists the same state when the page reloads. Will update

**Review by rightsaidjames — APPROVED** [review]
Looks good. I'm assuming the illustration to the right of the heading and copy text (like in other tabs) is still to be added in?

**Review by kpopper — APPROVED** [review]

**Pull request merged**